packages/integration-tests/cmr.js

```diff
@@ -159,32 +159,6 @@ async function waitForConceptExistsOutcome(cmrLink, expectation) {
   }
 }
 
-/**
- * Get the online resource links from the CMR objects
- *
- * @param {string} cmrLink - CMR URL path to concept,
- * i.e. what is returned from post to cmr task
- * @returns {Array<Object>} Array of link objects in the format
- * { inherited: true,
-    rel: 'http://esipfed.org/ns/fedsearch/1.1/metadata#',
-    hreflang: 'en-US',
-    href: 'https://opendap.cr.usgs.gov/opendap/hyrax/MYD13Q1.006/contents.html' }
- */
-async function getOnlineResources(cmrLink) {
```

Comment:
Removed as dupe function declaration.    Assuming it was inadvertent in 

https://github.com/nasa/cumulus/pull/824/files